ui/components/ui/token-input/token-input.component.js

```diff
@@ -34,6 +35,7 @@ export default class TokenInput extends PureComponent {
       symbol: PropTypes.string,
     }).isRequired,
     tokenExchangeRates: PropTypes.object,
+    tokens: PropTypes.array,
```

Comment:
I think we need this to be `required` since we're iterating over it on line 117.

Reply:
@darkwing Done [here](https://github.com/MetaMask/metamask-extension/pull/14713/commits/cc0a033db731d6444af9f2d59e95c7587a5971bf)